crates/router/src/core/refunds.rs

```diff
@@ -1113,3 +1114,63 @@ pub async fn retry_refund_sync_task(
         }
     }
 }
+
+#[instrument]
+pub fn payment_intent_to_auto_refund_struct(
+    payment_intent: PaymentIntent,
+    retry_count: i32,
+    max_retries: i32,
+) -> refund::AutoRefundWorkflow {
+    refund::AutoRefundWorkflow {
+        payment_intent,
+        retry_count,
+        max_retries,
+    }
+}
```

Comment:
You can do `impl From<PaymentIntent> for refund::AutoRefundWorkflow` for this

Reply:
But I need those two count variables also right ?